src/backend/columnar/README.md

```diff
@@ -0,0 +1,373 @@
+extension's source code directory. This will clean up all the files copied during
+the installation:
+
+    $ sudo PATH=/usr/local/pgsql/bin/:$PATH make uninstall
+
+
+Changeset
+---------
+### Version 1.7.0
+* (Fix) Add support for PostgreSQL 12
+* (Fix) Support count(t.*) from t type queries
+* (Fix) Build failures for MacOS 10.14+
+* (Fix) Make foreign scan parallel safe
+* (Fix) Add support for PostgreSQL 11 COPY
+### Version 1.6.2
+* (Fix) Add support for PostgreSQL 11
+### Version 1.6.1
+* (Fix) Fix crash during truncate (Cstore crashing server when enabled, not used)
+* (Fix) No such file or directory warning when attempting to drop database
+### Version 1.6
+* (Feature) Added support for PostgreSQL 10.
+* (Fix) Removed table files when a schema, extension or database is dropped.
+* (Fix) Removed unused code fragments.
+* (Fix) Fixed incorrect initialization of stripe buffers.
+* (Fix) Checked user access rights when executing truncate.
+* (Fix) Made copy command cancellable.
+* (Fix) Fixed namespace issue regarding drop table.
+
+### Version 1.5.1
+* (Fix) Verify cstore_fdw server on CREATE FOREIGN TABLE command
+
+### Version 1.5
+* (Feature) Added support for PostgreSQL 9.6.
+* (Fix) Removed table data when cstore_fdw table is indirectly dropped.
+* (Fix) Removed unused code fragments.
+* (Fix) Fixed column selection logic to return columns used in expressions.
+* (Fix) Prevented alter table command from changinf column type to incompatible types.
+
+### Version 1.4.1
+
+* (Fix) Compatibility fix for Citus [copy command][copy-command].
+
+### Version 1.4
+
+* (Feature) Added support for ```TRUNCATE TABLE```
+* (Fix) Added support for PostgreSQL 9.5
+
+### Version 1.3
+
+* (Feature) Added support for ```ALTER TABLE ADD COLUMN``` and ```ALTER TABLE DROP COLUMN```.
+* (Feature) Added column list support in ```COPY FROM```.
+* (Optimization) Improve row count estimation, which results in better plans.
+* (Fix) Fix the deadlock issue during concurrent inserts.
+* (Fix) Return correct result when using whole row references.
+
+### Version 1.2
+
+* (Feature) Added support for ```COPY TO```.
+* (Feature) Added support for ```INSERT INTO cstore_table SELECT ...```.
+* (Optimization) Improved memory usage.
+* (Fix) Dropping multiple cstore tables in a single command cleans-up files
+  of all them.
+
+### Version 1.1
+
+* (Feature) Make filename option optional, and use a default directory inside
+  $PGDATA to manage cstore tables.
+* (Feature) Automatically delete files on DROP FOREIGN TABLE.
+* (Fix) Return empty table if no data has been loaded. Previously, cstore_fdw
+  errored out.
+* (Fix) Fix overestimating relation column counts when planning.
+* (Feature) Added cstore\_table\_size(tablename) for getting the size of a cstore
+  table in bytes.
+
+
+Copyright
+---------
+
+Copyright (c) 2017 Citus Data, Inc.
+
+This module is free software; you can redistribute it and/or modify it under the
+Apache v2.0 License.
```

Comment:
We should remove this (maybe just remove this file?)

Reply:
Yeah, was an oversight to leave the readme, have removed now, after scanning there was nothing of interest to keep.

I will rewrite the history so that it gets removed right after the import once the PR is in a state we can land